config/accounts.go

```diff
@@ -12,10 +12,10 @@ type IntegrationType string
 
```

Comment:
I think we should change this to `type ChannelType string` since `IntegrationType` will eventually go away.